### PR TITLE
Release 0.8.2

### PR DESCRIPTION
**Pull request by jrbourbeau**
Closes https://github.com/nsidc/earthaccess/issues/404

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,16 @@
 # Changelog
 
-## Unreleased
-
+## [v0.8.2] 2023-12-06
 * Bug fixes:
-    * Follow Amazon EC2 IMDSv2 requirements to set `running-in-aws`
+    * Enable AWS check with IMDSv2
+    * Add region to running in AWS check
```
Comment (MattF-NSIDC) [inline CHANGELOG.md:6]: There was a breaking API change here, right? We changed `.am_i_in_aws()` to `.in_region()` I think, and I think we should call that out.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
+    * Handle opening multi-file granules
```
Comment (MattF-NSIDC) [inline CHANGELOG.md:7]: I didn't get a chance to look at this, but does it add any new surface to the API that we should call out by name, e.g. new method?

**Review by MattF-NSIDC — CHANGES_REQUESTED** [review]
Would like a little more detail in the changelog before you merge if you don't mind :)

File: CHANGELOG.md (hunk 1 above)
Comment (jrbourbeau) [reply CHANGELOG.md:6]: `_am_i_in_aws()` is a private method and my understanding is the `.running_on_aws` attribute is intended to be private (from @betolink in the offline `earthaccess` Slack channel). This is intended to just be an internal change, not a public-facing one.

Comment (MattF-NSIDC) [reply CHANGELOG.md:6]: :heavy_check_mark: thanks, just wanted to be completely sure. Without the leading underscore, I assumed :)

Comment (jrbourbeau) [reply CHANGELOG.md:6]: No worries -- same here

**Pull request merged**